modelmapper/cleaner.py

```diff
@@ -44,10 +44,22 @@ def __init__(self, *args, **kwargs):
         # https://github.com/python-excel/xlrd/blob/master/xlrd/xldate.py
         # 0: 1900-based, 1: 1904-based.
         self.xls_date_mode = kwargs.pop('xls_date_mode', 0)
+        self._missing_fields = set()
+        self.publicized = False
 
         super().__init__(*args, **kwargs)
 
-    def get_csv_data_cleaned(self, path_or_content, original_content_type=None):
+    def slack_missing_fields(self):
+        self.slack(
+            f'''Field found in a report where not defined in its given Modelmapping.
+                Fields Missing: {self._missing_fields}
+                Model: {self.settings.combined_file_name[:-3]}
+             '''
+        )
+
+    def get_csv_data_cleaned(self, path_or_content,
+                             original_content_type=None,
+                             ignore_new_fields=True):
```

Comment:
nit: newlines here seem unnecessary and inconsistent. If it is because of line width, should have:
```python
def get_csv_data_cleaned(self, path_or_content, original_content_type=None,
                         ignore_new_fields=True):
```